src/components/SigningModals/prop-types.js

```diff
@@ -0,0 +1,7 @@
+import PropTypes from 'prop-types'
+
+export const ModalProps = PropTypes.shape({
```

Comment:
Is this ModalProps used?

Reply:
Not yet but will be, it's for typing the render props when passed to a "screen"